docs/ref/models/querysets.txt

```diff
@@ -2474,6 +2474,51 @@ each model instance (if the database normally supports it).
     In older versions, enabling the ``update_conflicts`` parameter prevented
     setting the primary key on each model instance.
 
+.. versionadded:: 5.1
+
+    The ``where_clause`` parameter was added.
+
+    The ``where_clause`` parameter allows specifying a conditional SQL ``WHERE`` 
+    clause for the ``DO UPDATE SET`` part of the bulk insert query. This feature
```

Comment:
`DO UPDATE SET` is PostgreSQL-specific, so this should be reworded in a database-agnostic manner.